pkg/platform/runtime/buildscript/buildscript.go

```diff
@@ -109,6 +111,9 @@ func (v *Value) String() string {
 	case v.Str != nil:
 		return *v.Str
 
+	case v.Number != nil:
+		return strconv.FormatFloat(*v.Number, 'G', -1, 64)
```

Comment:
Could you please add a comment explaining the format values? Just to help avoid having to look up what these mean when we revisit this code.